packages/core/parcel/src/cli.js

```diff
@@ -230,7 +230,7 @@ async function run(entries: Array<string>, command: any) {
               ),
             500,
           );
-          await exit(1);
+          await exit();
```

Comment:
This seems to run no this way matter how Parcel was invoked (`serve`, `build`, etc.). I don't think it's a good idea to exit with a successful exit code when interrupting Parcel when used with `build` (could be a production build). 

Can we retain the non-zero code when interrupting `parcel build`?